tests/trestle/core/commands/author/command_test.py

```diff
@@ -25,60 +25,46 @@
         with pytest.raises(SystemExit) as wrapped_error:
             trestle.cli.run()
             # FIXME: Needs to be changed once implemented.
-            assert wrapped_error == SystemExit
-            assert wrapped_error.code == 1
+        assert wrapped_error.type == SystemExit
+        assert wrapped_error.value.code == 0
 
 
 def test_governed_docs_cli(tmp_trestle_dir: pathlib.Path) -> None:
     """Test happy path of md governed-docs subcommand."""
-    command = 'trestle md governed-docs'
+    command = 'trestle author docs'
     with mock.patch.object(sys, 'argv', command.split()):
         with pytest.raises(SystemExit) as wrapped_error:
             trestle.cli.run()
             # FIXME: Needs to be changed once implemented.
-            assert wrapped_error == SystemExit
-            assert wrapped_error.code == 1
+        assert wrapped_error.type == SystemExit
+        assert wrapped_error.value.code == 2
 
 
 def test_governed_folders_cli(tmp_trestle_dir: pathlib.Path) -> None:
-    """Test happy path of md governed-folders subcommand."""
-    command = 'trestle md governed-folders'
+    """Test happy path of author governed-folders subcommand."""
+    command = 'trestle author folders'
     with mock.patch.object(sys, 'argv', command.split()):
         with pytest.raises(SystemExit) as wrapped_error:
             trestle.cli.run()
             # FIXME: Needs to be changed once implemented.
-            assert wrapped_error == SystemExit
-            assert wrapped_error.code == 1
-
-
-def test_governed_projects_cli(tmp_trestle_dir: pathlib.Path) -> None:
-    """Test happy path of md governed-projects subcommand."""
-    command = 'trestle md governed-projects'
-    with mock.patch.object(sys, 'argv', command.split()):
-        with pytest.raises(SystemExit) as wrapped_error:
-            trestle.cli.run()
-            # FIXME: Needs to be changed once implemented.
-            assert wrapped_error == SystemExit
-            assert wrapped_error.code == 1
+        assert wrapped_error.type == SystemExit
+        assert wrapped_error.value.code == 2
 
 
 @pytest.mark.parametrize(
     'command_string',
-    [
-        ('trestle md governed-docs setup -tn test'), ('trestle md governed-folders setup-tn test'),
-        ('trestle md cidd setup')
-    ]
+    [('trestle author docs setup -tn test'), ('trestle author folders setup -tn test'), ('trestle author cidd')]
 )
 def test_failure_not_trestle(command_string, tmp_path: pathlib.Path) -> None:
     """Test for failure based on not in trestle directory."""
     with mock.patch.object(sys, 'argv', command_string.split()):
         with pytest.raises(SystemExit) as wrapped_error:
             trestle.cli.run()
             # FIXME: Needs to be changed once implemented.
-            assert wrapped_error == SystemExit
-            assert wrapped_error.code == 1
+        assert wrapped_error.type == SystemExit
+        assert wrapped_error.value.code == 1
```

Comment:
Wow - it was good to catch this.  A lot of them...

Reply:
Yes it did cause some problems.. a bit of a nasty find.